implementations/rust/daemon/src/config.rs

```diff
@@ -16,6 +16,11 @@ pub enum Input {
     Stdin,
 }
 
+#[derive(Debug, Clone)]
+pub enum AddonKind {
```

Comment:
Should this be an empty trait similar to vault?

Reply:
We should consider that if we intended to ship `ockamd` as a lib, but it's only going to be downloaded as a pre-built binary (or build from source).